### PR TITLE
Readme getting started content update

### PR DESCRIPTION
**Pull request by rajivsam**
Udated the getting started section with the details of running tensorflow and torch examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,26 +50,44 @@ When data scientists have refined their models to a point where they are ready t
...
 
     ` docker run -p 6529:8529 -p 8888:8888 -p 3000:3000 -it arangopipe/ap_tensor_flow`  (_tensorflow_)
     
     or, for _torch_,
     
     ` docker run -p 6529:8529 -p 8888:8888 -p 3000:3000 -it arangopipe/ap_torch`
 
+
+
+3. Running an example in the *torch* container: Run the *torch* docker container. You will have to use the `docker ps` command and get to the shell in the container using the `docker exec` command. Change directory to the `examples/pytorch` directory. The *torch* container provides an example of a linear regression model that uses **Arangopipe** to log experiment metadata. The experiment meta data includes information about the dataset, featureset and optimization settings used to run the *pytorch* model. Once you are in the shell of the *torch* container, run the driver program that develops the torch model and logs the experiment meta-data to *arangopipe*. The details are shown in the figure below.
```
Comment (joerg84) [inline README.md]: Can you provide the actual commands here?

File: README.md [hunk 2]
```diff
@@ -50,26 +50,44 @@ When data scientists have refined their models to a point where they are ready t
...
+
+ 
+    <img src="run_torch_driver.png" height="400">
+
+4. Execute this step after the model development step above has completed. Point your browser http:localhost:3000. Login to the Arangopipe user interface with username  root and password  `open sesame`. Select `Models` in the `Search Metadata` content pane. You should see the model you developed in the previous step. The details are shown in the figure below.
```
Comment (joerg84) [inline README.md]: http://

File: README.md [hunk 3]
```diff
@@ -111,26 +134,12 @@ Arangopipe represents metadata as a graph. The nodes of the graph above are the
...
-
-    <!--- ![Test Data Generation](test_data_generation.png) --->
-    <img src="test_data_generation.png" height="400">
-
-    You can then point your browser to `http://localhost:8888`. When prompted for a password, enter     `root` You will then see an examples notebook that explains how the examples are laid out in the directory. The examples illustrate how arangopipe can work with various tools in a machine learning stack. Examples of integration with _hyperopt_, _scikitlearn_, _tensorflow_, _torch_ and _mlflow_ are provided. 
+## Installing Arangopipe API
+To install the **Arangopipe** API, you would need to do the following:
 
-3. To use the **Arangopipe** user interface, point your browser to `http://localhost:3000`. To sign into the user interface, use a username of `root` and `open sesame` as password (note, there is a space between open and sesame). You neeed to complete the steps above to use the user interface. The features of the user interface are provided in the next section.
+1. `pip install python-arango == 4.4.0`
```
Comment (joerg84) [inline README.md]: Can't we preinstall that in the image?

File: README.md (hunk 1 above)
Comment (rajivsam) [reply README.md]: Sure the actual commands are in the screen shot below, but I can add the commands there.

File: README.md (hunk 2 above)
Comment (rajivsam) [reply README.md]: ok, will fix that.

File: README.md (hunk 3 above)
Comment (rajivsam) [reply README.md]: Actually this is pre-installed in the docker image. This is incase someone wants to use the API outside of the docker image. If you think that is confusing, I will just remove the whole install section.

**Pull request merged**